app/scripts/modules/google/src/loadBalancer/configure/internal/gceCreateInternalLoadBalancer.controller.ts

```diff
@@ -250,6 +252,7 @@ module(GCE_INTERNAL_LOAD_BALANCER_CTRL, [
     GCE_COMMON_LOAD_BALANCER_COMMAND_BUILDER,
     ACCOUNT_SERVICE,
     INFRASTRUCTURE_CACHE_SERVICE,
+    require('../../../common/xpnNaming.gce.service.js'),
```

Comment:
`require('google/common/xpnNaming.gce.service.js')` would be a little easier to read.